GitOut/Features/Wpf/Converters/ColorToBrushConverter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace GitOut.Features.Wpf.Converters
+{
+    public class ColorToBrushConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+            => value is Color color ? new SolidColorBrush(color) : DependencyProperty.UnsetValue;
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+            => throw new NotSupportedException();
```

Comment:
Should return `UnsetValue` or `Binding.DoNothing` according to https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.ivalueconverter.convertback?view=netframework-4.8